shuriken/lib/parser/Macho/macho_header.cpp

```diff
@@ -0,0 +1,32 @@
+//--------------------------------------------------------------------*- C++ -*-
+// Shuriken-Analyzer: library for bytecode analysis.
+// @author Farenain <kunai.static.analysis@gmail.com>
+//
+// @file macho_header.cpp
+
+#include "shuriken/parser/Macho/macho_header.h"
+#include "shuriken/common/logger.h"
+
+using namespace shuriken::parser::macho;
+
+void MachoHeader::parse_header(common::ShurikenStream &stream) {
+
+    log(LEVEL::INFO, "Start parsing header");
+
+    // read the macho header
+    stream.read_data<machoheader_t>(machoheader, sizeof(machoheader_t));
+
+    // print the macho header
```

Comment:
a `parse_*` function should not contain the printing of the fields, this corresponds to another function where the information is dumped, but if you add it here, the data will be printed every time you parse any mach-o header.

Reply:
That's right, I'll divide it into two functions and add some checkings. Thanks!